### PR TITLE
fix: don't charge zero fee

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: test/CSModule.t.sol [hunk 1]
```diff
@@ -3119,20 +3119,51 @@ contract CsmRemoveKeys is CSMCommon {
...
+        vm.recordLogs();
+
+        vm.prank(nodeOperator);
+        csm.removeKeys(noId, 1, 2);
+
+        assertEq(
+            vm.getRecordedLogs().length,
+            4,
+            "should not emit events related to charge fee"
+        );
```
Comment (dgusakov) [inline test/CSModule.t.sol:3166]: Can we make it more explicit? Now it is just count and no explicit check of the exact event

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: src/CSModule.sol [hunk 2]
```diff
@@ -1637,9 +1637,12 @@ contract CSModule is
         // The Node Operator is charged for the every removed key. It's motivated by the fact that the DAO should cleanup
         // the queue from the empty batches related to the Node Operator. It's possible to have multiple batches with only one
         // key in it, so it means the DAO should be able to cover removal costs for as much batches as keys removed in this case.
-        uint256 amountToCharge = keyRemovalCharge * keysCount;
-        accounting.chargeFee(nodeOperatorId, amountToCharge);
-        emit KeyRemovalChargeApplied(nodeOperatorId, amountToCharge);
+        uint256 removalFee = keyRemovalCharge;
```
Comment (madlabman) [inline src/CSModule.sol:1640]: If we calculate `amountToCharge` in advance we save space on the stack, I guess.

Comment (vgorkavenko) [reply src/CSModule.sol]: done

File: test/CSModule.t.sol (hunk 1 above)
Comment (vgorkavenko) [reply test/CSModule.t.sol]: done

**Review by dgusakov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**